source/connect.txt

```diff
@@ -267,14 +267,11 @@ To disconnect from a deployment and exit ``mongosh``, you can:
 Limitations
 -----------
 
-- ``mongosh`` has the following 
-  :manual:`Kerberos authentication </core/kerberos/>` limitations:
-
-  - ``authMechanismProperties=CANONICALIZE_HOST_NAME:true|false`` is not
-    permitted in the connection string. Instead, specify hostname 
-    canonicalization in the connection string with 
-    ``authMechanismProperties=gssapiCanonicalizeHostName:true|false``.
-  - Hostname canonicalization reverse lookup is not supported.
+- :manual:`Kerberos authentication </core/kerberos/>` does not permit
+  ``authMechanismProperties=CANONICALIZE_HOST_NAME:true|false`` in the
+  connection string. Use one of:
```

Comment:
Even this isn’t quite accurate, because you *can* use the legacy names, but I think documenting it as if it didn’t work is also just fine :slightly_smiling_face:

Reply:
OK, I'll leave it as is. OK to push the PR?

Reply:
Oops, just saw the LGTM